static/index.scss

```diff
@@ -47,16 +47,22 @@
     margin-top: 1em;
     padding: 0 .5em;
     border-right: 4px solid black;
+    a {
+        text-decoration: none;
+        color: rgba($links,1);
```

Comment:
You can just do $links.
rgba($color, x) adjusts the `a`lpha value of the color, AKA the opacity, AKA see through-ness. So 1 is completely opaque (the default) and 0 is completely invisible.